lib/identity/parsers/gc_ids_constants.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+module Identity
+  module Parsers
+    module GCIdsConstants
+      # Originating IDs, defines what underlying system the ID has been sourced from
+      VA_ROOT_OID = '2.16.840.1.113883.4.349'
+      DOD_ROOT_OID = '2.16.840.1.113883.3.42.10001.100001.12'
+
+      # The follow set of regex match each full ID format and extract the specific ID value from each
+      # They tend to follow the format: <icn>^<id_type>^<assigning_authority>^<assigning_facility>^<id_state>
+
+      # ICN_REGEX, ex. 16701377^NI^200M^USVHA^A
```

Comment:
Can we add an empty line in between these to make it a little easier to read?

Reply:
👍  Sounds good